packages/collector/test/tracing/messaging/kafka-avro/producer.js

```diff
@@ -0,0 +1,80 @@
+/*
+ * (c) Copyright IBM Corp. 2022
+ */
+
+'use strict';
+
+require('../../../../')({
+  tracing: {
+    kafka: {
+      headerFormat: 'string'
```

Comment:
Notes from call:

Because of the [binary bug in rdkafka](https://github.com/Blizzard/node-rdkafka/pull/935) the span correlation wouldn't work at all in binary mode (which is still the default mode in kafka instrumention)

Node.js tracer has already added support for kafka string headers although the kafka header migration has not even started yet (or is not finished yet).
Setting the `headerFormat` to 'string' here means, rdkafka/avro instrumention works if the consumer tracer and the producer tracer support and use string format.

Therefor the customer needs to set `headerFormat: string`.

@willianpc as discussed: `headerFormat` is not documented yet in our public docs. We might want to wait for @basti1302 opinion here

Reply:
The point was not to remove something in the PR :)

How can the customer use kafka-avro when there is no general docs about headerFormat?
Refs https://github.ibm.com/instana/docs/pull/949/files

Let's discuss offline :)

Reply:
I will push the change from the Go side as well. The card is sitting there for a while.